lib/render/VolumeRenderer.cpp

```diff
@@ -391,21 +394,31 @@ int VolumeRenderer::_initializeAlgorithm()
 int VolumeRenderer::_loadData()
 {
     VolumeParams *RP = (VolumeParams *)GetActiveParams();
+    vector <double> minExt, maxExt;
+    RP->GetBox()->GetExtents(minExt, maxExt);
+
     CheckCache(_cache.var, RP->GetVariableName());
     CheckCache(_cache.ts, RP->GetCurrentTimestep());
     CheckCache(_cache.refinement, RP->GetRefinementLevel());
     CheckCache(_cache.compression, RP->GetCompressionLevel());
+    CheckCache(_cache.minExt, minExt);
+    CheckCache(_cache.maxExt, maxExt);
```

Comment:
This will cause the data to be reloaded every time that the user changes the extents. Could we check to see if the new extents are still within the previously loaded grid and only reload the data if needed?